src/components/HelpMenu.tsx

```diff
@@ -25,8 +25,9 @@ const HelpMenu = ({ ...rest }: HelpMenuProps) => {
   const feedbackDisclosure = useDisclosure();
   const intl = useIntl();
   const MenuButtonRef = useRef(null);
+  const ContainerRef = useRef(null);
```

Comment:
Can we lower case the ref names?

Reply:
[52ab443](https://github.com/microbit-foundation/ml-trainer/pull/488/commits/52ab44347e3f1543ea14c8c3d4e788cf0428fcfc)